package.json

```diff
@@ -102,7 +104,10 @@
   "browserslist": "chrome 70, firefox 70, safari 11.1",
   "prettier": {
     "arrowParens": "avoid",
-    "singleQuote": true
+    "singleQuote": true,
+    "importOrder": ["^[./]"],
+    "importOrderSeparation": true,
+    "importOrderCaseInsensitive": true
```

Comment:
I would be inclined to leave this at the default `false` value, just because that lines up with what happens when eg. sorting identifiers in an object using an editor's sort command.